src/Hedgehog/Journal.fs

```diff
@@ -0,0 +1,24 @@
+namespace Hedgehog
+
+[<Struct>]
+type Journal =
+    | Journal of seq<unit -> string>
+
+module Journal =
+    let ofList (xs : seq<unit -> string>) : Journal =
+        Journal xs
+
+    let toList (Journal xs : Journal) : List<string> =
```

Comment:
How about calling this `eval` and returning `seq` instead of `list`?

Reply:
Done